test_common/harness/imageHelpers.cpp

```diff
@@ -484,6 +484,28 @@ size_t compare_scanlines(const image_descriptor *imageInfo, const char *aPtr,
             }
             break;
 
+            case CL_SNORM_INT8: {
+                cl_uchar aPixel = *(cl_uchar *)aPtr;
+                cl_uchar bPixel = *(cl_uchar *)bPtr;
+                // -1.0 is defined as 0x80 and 0x81
+                if ((aPixel != bPixel) && ((aPixel | bPixel) != 0x81))
+                {
+                    return column;
+                }
+            }
+            break;
+
+            case CL_SNORM_INT16: {
+                cl_ushort aPixel = *(cl_ushort *)aPtr;
+                cl_ushort bPixel = *(cl_ushort *)bPtr;
+                // -1.0 is defined as 0x8000 and 0x8001
+                if ((aPixel != bPixel) && ((aPixel | bPixel) != 0x8001))
```

Comment:
See above, and apply whatever fix we make for `CL_SNORM_INT8` to `CL_SNORM_INT16` here.

Reply:
Nice catch, thanks. I like your suggestion.